cmd/fleet/cron.go

```diff
@@ -643,6 +643,9 @@ func newWorkerIntegrationsSchedule(
 			}
 			return nil
 		}),
+		schedule.WithJob("dep_cooldowns", func(ctx context.Context) error {
+			return worker.ProcessDEPCooldowns(ctx, ds, logger)
+		}),
```

Comment:
am I getting this right and this will potentially run every 1 minute? I thought this was in the cleanups cron. I'm curious if 1 hour was too slow?

It's probably fine as we don't expect many results in the tables this jobs queries, but I'm curious.

Reply:
You've got it right. It wasn't that cleanups were too slow necessarily, but this cron was already dealing with the worker so I thought it made sense to locate this here. Because we're indexing the response in `host_dep_assignments` and filter on the `FAILED` state (which we expect to be pretty a rare occurence), it seems like this wouldn't be very taxing on the DB. And FWIW this way we avoid the scenario where cooldown expires just after the hourly cron so it ends up being nearly two hours for the retry.